src/ophys_etl/modules/denoising/fine_tuning/utils/data_splitter.py

```diff
@@ -20,9 +22,14 @@ def __init__(
             Path to movie
         seed
             Seed for reproducibility
+        downsample_frac
+            Amount to downsample the data by. I.e. a downsample frac
+            of 0.1 would reduce the dataset by 10%. Default is no downsampling
```

Comment:
Maybe change this to "reduce the dataset size by 10%"? Here and the previous mention that is.